src/oc_erchef/apps/data_collector/src/data_collector.erl

```diff
@@ -50,3 +44,21 @@ is_enabled() ->
         undefined ->
             false
     end.
+
+-spec token() -> list() | atom().
+token() ->
+    case application:get_env(data_collector, token) of
+        {ok, Token} ->
+            Token;
+        undefined ->
+            false
+    end.
+
+-spec log_skipped_actions() -> boolean().
+log_skipped_actions() ->
+    case application:get_env(data_collector, log_skipped_actions) of
```

Comment:
Since we are returning a boolean, you might consider changing the item we render to the config file the atom `true` and then making this:

```
log_skipped_actions() ->
   application:get_env(data_collector, log_skipped_actions, false).
````